daemon/api_test.go

```diff
@@ -1346,9 +1272,9 @@ func (s *apiSuite) TestLoginUserBadRequest(c *check.C) {
 }
 
 func (s *apiSuite) TestLoginUserDeveloperAPIError(c *check.C) {
-	mockDeveloperAPIServer := s.makeDeveloperAPIServer(200, "{}")
```

Comment:
The low level bits are also tested in `store_test.go:TestLoginUserDeveloperAPIError()`